### PR TITLE
add pagination for workflows 

### PR DESCRIPTION
**Pull request by reillyse**
# Description

- [x] New feature (non-breaking change which adds functionality)
\
## What's Changed

Added pagination for workflows and extended workflow list to 200 workflows

### CODE REVIEW

File: frontend/app/src/pages/main/workflows/components/workflow-table.tsx [hunk 1]
```diff
@@ -40,18 +41,51 @@ export function WorkflowTable() {
...
+  ]);
+
   const data = useMemo(() => {
+
     const data = listWorkflowQuery.data?.rows || [];
+    setPageIndex(listWorkflowQuery.data?.pagination?.num_pages || 0);
 
     return data;
-  }, [listWorkflowQuery.data?.rows]);
+  }, [listWorkflowQuery.data?.rows , searchParams]);
```
Comment (grutt) [inline frontend/app/src/pages/main/workflows/components/workflow-table.tsx]: this wont be autofixed with lint:fix

File: pkg/repository/prisma/dbsqlc/workflows.sql [hunk 2]
```diff
@@ -82,7 +82,8 @@ SELECT
 FROM
     "Workflow" as workflows
 WHERE
-    workflows."tenantId" = $1 AND
+    workflows."tenantId" = @tenantId::uuid AND
+    workflows."name" LIKE '%' || @name::text || '%' AND
```
Comment (grutt) [inline pkg/repository/prisma/dbsqlc/workflows.sql]:
```suggestion
    ( @name::text  IS NULL OR workflows."name" LIKE '%' || @name::text || '%' AND)
```
We should null check the input first,
we might need to use sqlc.nargs here

Comment (grutt) [reply pkg/repository/prisma/dbsqlc/workflows.sql]:
```
AND
        (
            sqlc.narg('search')::text IS NULL OR
            workflow.name like concat('%', sqlc.narg('search')::text, '%')
        )
```

Also, curious how SQLC interprets the `||` is it the same as concat?

**Review by grutt — CHANGES_REQUESTED** [review]
lint required

File: frontend/app/src/pages/main/workflows/components/workflow-table.tsx (hunk 1 above)
Comment (reillyse) [reply frontend/app/src/pages/main/workflows/components/workflow-table.tsx]: linted

File: pkg/repository/prisma/dbsqlc/workflows.sql (hunk 2 above)
Comment (reillyse) [reply pkg/repository/prisma/dbsqlc/workflows.sql]: || is just string concatenation  - standardized it to your way

**Review by grutt — APPROVED** [review]
ship it!

**Pull request merged**